android/app/build.gradle

```diff
@@ -23,6 +23,7 @@ if (flutterVersionName == null) {
 
 apply plugin: 'com.android.application'
 apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.firebase.crashlytics'
```

Comment:
for example, I do not added any dependencies in to native, just flutter fire configure =)

Reply:
![image](https://user-images.githubusercontent.com/111364799/195439852-21df5e74-a6a2-43c4-ae69-d3cd6bf15567.png)

maybe someday it will come back to haunt you)

Reply:
Great, but this is android doc. Here is flutter
https://firebase.google.com/docs/crashlytics/get-started?hl=en&authuser=0&platform=flutter

Reply:
0_о I spent half a day with kotlin, already with one foot in the native :)